test/unit/staticWebsite.test.ts

```diff
@@ -187,11 +192,19 @@ describe("static website", () => {
         });
     });
 
-    it("should support custom domains", async () => {
+    it("should support a custom domain", async () => {
         const { cfTemplate } = await runServerless({
-            fixture: "staticWebsiteDomain",
-            configExt: pluginConfigExt,
             cliArgs: ["package"],
+            config: Object.assign(baseConfig, {
+                "static-website": {
+                    landing: {
+                        path: ".",
+                        domain: "example.com",
```

Comment:
Tests fail here: Serverless does not validate this. Maybe my JSON schema is wrong?